pkgs/development/python-modules/digitalocean/default.nix

```diff
@@ -40,6 +40,8 @@ buildPythonPackage rec {
     cd digitalocean
   '';
 
+  disabledTests = ["TestFirewall"];
```

Comment:
maybe add a comment why the test is disabled?

Reply:
Done.